src/main/java/teammates/common/util/Const.java

```diff
@@ -286,6 +295,8 @@ private Const() {
         public static final String SESSION_LINKS_RECOVERY = URI_PREFIX + "/sessionlinksrecovery";
         public static final String NATIONALITIES = URI_PREFIX + "/nationalities";
         public static final String EMAIL = URI_PREFIX + "/email";
+        public static final String TRACK_SESSION = URI_PREFIX + "/track/session";
+        public static final String SESSION_LOGS = URI_PREFIX + "/logs/session";
```

Comment:
Would like to ask for feedback on the API endpoint. Are the names appropriate? Perhaps we could also remove `/track/session` and preserve only `/logs/session` -- such that a POST request to `/logs/session` would be for creating a log, and a GET request to it would be for searching for logs, so there's no need for two different names. Would it be better?

Reply:
You're right, I think a single endpoint `/logs/session` would be much more appropriate. Resource endpoint names should be nouns so `/track/session` is already very questionable. Can you help to make this change? Should just take a little tweaking